appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -479,6 +483,13 @@ private boolean writeAndroidManifest(File manifestFile, Set<String> permissionsN
       }
 
       out.write("  </application>\n");
+
+      // This might be necessary to get Google Play to show the app to tablets, on some devices.
+      // It does not, however, get rid of the "designed for phones" comment in Play.  That requires
+      // API (11?) and adding additionaal resource folders for large screens.
+      out.write ("<supports-screens android:largeScreens=\"true\" android:xlargeScreens=\"true\" android:anyDensity=\"true\" /> \n");
```

Comment:
I'm not sure what this will do... I'm going to remove it and merge the change above (the one dealing with the keyboard)